app/ts/schema.ts

```diff
@@ -50,8 +51,12 @@ export function createUnitParser(decimals?: bigint): funtypes.ParsedValue<funtyp
 
 export const AddressParser: funtypes.ParsedValue<funtypes.String, string>['config'] = {
 	parse: value => {
-		if (!isAddress(value)) return { success: false, message: `${value} is not a valid address string.` }
-		else return { success: true, value: getAddress(value) }
+		try {
+			// convert hex string to bigint to strip possible leading zeros
+			const maybeAddress = value && BigInt(value).toString(16).padStart(40, '0')
+			if (isAddress(maybeAddress)) return { success: true, value: getAddress(maybeAddress) }
+		} catch (error) { }
```

Comment:
Why are you swallowing errors here?  Feels like errors should be surfaced and fixed rather than silently dropped.

Reply:
the try/catch will catch some `BigInt` thrown errors, which should always be due to value being other than a valid address. The next line (from the highlighted) will return the the invalid result object.

Reply:
Just to get in writing what we discussed in the meeting today: We shouldn't be using exceptions for control flow.  We should be validating the input here in a way that doesn't throw an error, because users inputting bad data is not exceptional, it is normal behavior/flow of the application.